### ISSUE TITLE
Keep getting 403 after installation 

### ISSUE THREAD

**Issue by Antoine-lb** [XanderD99/strapi-prometheus#6]
Hello 👋

**Describe the bug**
I followed the installation step by step but I keep getting a 403 error (Authenticated and public rol have access to the `/api/metric`)

I created a brand new repo to test and I got the same error:
[https://github.com/Antoine-lb/strapi-prom-test](https://github.com/Antoine-lb/strapi-prom-test)

Would you mind giving me a hint on what may be the problem?

This plugin may be a game changer for me, if I manage to make it work I would be very happy.

Thanks, have a great day!

**Comment by XanderD99** [comment]
Yes, right now you cannot use the roles as a way to secure that endpoint, partially due to the fact that there is little to no documentation (that I have found) about how the whole user-permissions plugin works. 
Therefor I chose to only enable use of the api token of strapi (this is enabled by default). So you should create an api token and use that instead of username/password for authentication. 

I am sorry for this inconvenience but it was the fastest way to enable auth. This may change in the future but I haven't had any time to work on this.

so TLDR;
Use api token (read-access) instead of username/password of user-permissions plugin to authenticate to the /api/metric endpoint.

**Comment by XanderD99** [comment]
I did update the readme as this wasn't mentioned there yet

**Comment by Antoine-lb** [comment]
Thanks a lot for your answer!
